src/groupelements.jl

```diff
@@ -0,0 +1,143 @@
+# Sectors corresponding to irreducible representations of compact groups
+#------------------------------------------------------------------------------#
+"""
+    abstract type AbstractGroupElement{G<:Group,ω} <: Sector end
```

Comment:
Is it useful to have the cocycle field parameter in the abstract type? I don't immediately see where it is used, and given that `cocycle` is access through a function it might not really be necessary to have that. (This comes back to the idea of working a bit more with traits than with abstract supertypes with too many type parameters)